qiskit_machine_learning/algorithms/classifiers/qsvc.py

```diff
@@ -51,7 +51,9 @@ def __init__(self, *args, quantum_kernel: Optional[QuantumKernel] = None, **kwar
 
         super().__init__(
             kernel=self._quantum_kernel.evaluate,
-            random_state=algorithm_globals.random_seed,
+            random_state=kwargs.get("random_state")
+            if "random_state" in kwargs
+            else algorithm_globals.random_seed,
```

Comment:
It maybe easier/simpler to say
```
if "random_state" not in kwargs:
  kwargs["random_state"] = algorithm_globals.random_seed
```
ahead of the super init  call as it was and just pass it up in kwargs rather than do so explicitly by name parameter where we may end up with passing it in kwargs as well if it was there already,

Reply:
@woodsp-ibm Thank you for the suggestion - I've edited the code.